aks-deploy/README.md

```diff
@@ -0,0 +1,41 @@
+ 
+# GitHub Action for deploying to Azure K8S Service
+
+
+## Usage
+
+```
+
+
+action "Deploy to Azure Kubernetes Service" {
+  uses = "actions/azure/aks@master"
+  secrets = ["DOCKER_PASSWORD", "KUBE_CONFIG_DATA"]
+  env = {
+    AKS_CLUSTER_NAME = "<AKS Cluster Name>"
+    DOCKER_USERNAME = "<Docker Registry username>"
+    DOCKER_REGISTRY_URL = "<Docker Registry URL>"
+    CONTAINER_IMAGE_NAME = "<Container Image Name>" 
+  }
+}
+
+```
+
+
+### Secrets
```

Comment:
Having them as secrets is a choice, I suppose you don't need to mandate this?

Reply:
Yes, its a way to recommend that these variables should be Secrets. It follows the same pattern as other GH actions today